### DISCUSSION TITLE
[Help]: Disable skipping slides by vigorous swipe

### DISCUSSION THREAD

**Discussion by romancavry** [davidjerleke/embla-carousel#888] [Help] [answered]
### Summary

The problem mostly with wheel gestures. I need to prevent skipping slides even if the swipe was vigorous. `skipSnaps: false` cannot help here. 

By the way, I had read [this comment](https://github.com/davidjerleke/embla-carousel/issues/38#issuecomment-565814315) and appreciate this behavior. But in certain cases it would be very helpful to make the carousel feel stiff by preventing skipping slides by wheel gestures. Even if the user drags the carousel with decent force.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://codesandbox.io/p/devbox/wild-https-wm7nzs

**Comment by davidjerleke** [comment]
@romancavry I agree that the swipe gestures plugin should honor the [`skipSnaps`](https://www.embla-carousel.com/api/options/#skipsnaps) option. But that plugin belongs to a [separate repository](https://github.com/xiel/embla-carousel-wheel-gestures) created by a different author so please create your issue there.

**Comment by romancavry** [comment, marked as answer]
For those who are also interested in this problem. 

https://github.com/xiel/embla-carousel-wheel-gestures/issues/201
